README.md

```diff
@@ -1,2 +1,143 @@
-# hello_world
-Repo created for excercise "Set up a 'Hello Microverse' project" of Microverse program, day 2
+    <a name="readme-top"></a>
+    
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello World Excercise</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello World] <a name="about-project"></a>
+
+**[Hello World]** is a project to resolve the exercise named "Set up a 'Hello Microverse' project" of Microverse program, day 2.
+
+## 🛠 Built With HTML and CSS <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The project was developed using HTML and CSS.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Saying hello to the world.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> www.helloworld.com
+
+- [Live Demo Link](https://www.helloworld.com)
```

Comment:
- [ ]  Kindly remove this template in order to have a clean file. 👍
- [ ] Please take note that the live demo should link to your page not an external page; if you don't have any live demo at the moment, you can just write "**The live link will be updated soon**". ℹ️

Reply:
The template was removed and the text was replaced as requested.